docs/gm-commands.md

```diff
@@ -1,6 +1,10 @@
----
-redirect_from: "/GM-Commands"
----
+# ENABLE GM POWERS
+
+To give yourself GM powers, you first need to enable this for your account. After creating a account and password to be able to login, apply the following after:
+Account set gmlevel accountname 3 -1 
+
+3 stands for the GM level (4 is max), -1 stands for ALL realms.
```

Comment:
```suggestion
To give yourself GM powers, you first need to enable this for your account. After creating an account and password to be able to login, apply the following:
`account set gmlevel accountname 3 -1`

3 stands for the GM level (4 is max), -1 stands for ALL realms.
```

Reply:
Nonono just link to the existing page if anything